the-collab-lab/tcl-16-smart-shopping-list
# 21. As a user, I want the "Add Items" page to look visually appealing

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description



Updated the layout / styles for AddItem page according to the mockup. I created custom radio buttons with css to resemble the ones in the mockup. 

## Related Issue


closes #45 

## Acceptance Criteria


style the "Add Items" page according to the mockup

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before


![Screen Shot 2021-02-25 at 7 39 30 PM](https://user-images.githubusercontent.com/46036289/109238597-cf856d00-77a1-11eb-8246-0f3fc1dd30cd.png)

### After


![Screen Shot 2021-02-25 at 7 38 58 PM](https://user-images.githubusercontent.com/46036289/109238622-dca25c00-77a1-11eb-80f2-846d85da0122.png)


## Testing Steps / QA Criteria


1. pull from branch `mr-add-items-page-ui`
2. npm start
3. navigate to Add Item page
4. check that the UI resembles mockup but is using the colours/fonts in the notion documentation (what we have in `variables.scss` )
